crypto/cmp/cmp_status.c

```diff
@@ -83,10 +79,8 @@ int ossl_cmp_pkisi_get_pkifailureinfo(OSSL_CMP_PKISI *si)
     int i;
     int res = 0;
 
-    if (si == NULL || si->failInfo == NULL) {
-        CMPerr(0, CMP_R_NULL_ARGUMENT);
-        return -1;
-    }
+    if (!ossl_assert(si != NULL && si->failInfo != NULL))
+        return 0;
```

Comment:
Fix introduced wrong error return code - should stay -1 here.

Reply:
Fixed.